common/src/main/java/com/wynntils/core/rendering/RenderManager.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.rendering;
+
+import com.wynntils.core.rendering.objects.RenderOptionHolder;
+import com.wynntils.core.rendering.objects.RenderPosition;
+import com.wynntils.core.rendering.objects.RenderableHolder;
+import com.wynntils.core.rendering.properties.RenderOption;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.reflect.FieldUtils;
+
+/**
+ * Managing class for Features that have a renderable element.
+ *
+ * Used to store all the renderable elements
+ */
+public class RenderManager {
```

Comment:
Perhaps name this `OverlayRenderManager` so it does not get confused with other types of rendering stuff (like `RenderUtils`)?

Reply:
I'm going with `OverlayManager` since it technically doesn't do any rendering